runfile/trafficgen/trafficgen-remotehosts-runfile.json

```diff
@@ -0,0 +1,107 @@
+{
+  "tags": {
+    "topology": "RHEL10-ctx7-test-002pct"
```

Comment:
Let's drop this or make it much more generic and not reference something RHEL specific.